### ISSUE TITLE
Separate overlapping avatars with 2dp

### ISSUE THREAD

**Issue by stefan-niedermann** [stefan-niedermann/nextcloud-deck#433]
The overlapping avatars should be separated from each other with a `2dp` border.

**Comment by stefan-niedermann** [comment]
cc @jancborchardt for the 2px border around the avatars.

![grafik](https://user-images.githubusercontent.com/4741199/80514584-01128980-8981-11ea-8685-4c8c415e1d95.png)
![grafik](https://user-images.githubusercontent.com/4741199/80514628-0d96e200-8981-11ea-8309-099c44d0a834.png)

**Comment by jancborchardt** [comment]
> cc @jancborchardt for the 2px border around the avatars.
> 
> ![grafik](https://user-images.githubusercontent.com/4741199/80514584-01128980-8981-11ea-8685-4c8c415e1d95.png)
> ![grafik](https://user-images.githubusercontent.com/4741199/80514628-0d96e200-8981-11ea-8309-099c44d0a834.png)
> 

Great stuff, looks perfect! :) Is this a component of sorts we could re-use in the Files app too?

**Comment by stefan-niedermann** [comment]
In theory this is reusable.

@tobiasKaminsky if you **want** to reuse this component, i will put some :heart: into it and extract it into a own module :package:.

But beware, the border is a hard color value which has to be set depending on where you are using the `OverlappingAvatarsLayout` - because in the sample case the background of the material card uses a transparent color, and if use the exact same color it will multiply and the the border-color will be darker than the background of the cards.

**TL;DR: Contact me in Telegram if you want to reuse it.**
